src/multiple_shooting.jl

```diff
@@ -0,0 +1,41 @@
+"""
+Returns the predictions on the whole ODE data and a total loss after trying a 'Direct multiple shooting' on ODE data.
+In Direct Multiple Shooting, the Neural Network divides the interval into smaller intervals and solves for them separately.
+The default group size is 5 implying the whole dataset would be divided in groups of 5 and the Neural Network will solve on them individually.
+The default continuity term is 100 implying any losses arising from the non-continuity of 2 different groups will be scaled by 100.
+
+```julia
+loss_neuralode_helper(p,ode_data,tsteps,prob,loss_function,grp_size=5,continuity_term=100)
+```
+Arguments:
+- `p`: The parameters of the Neural Network to be trained.
+- `ode_data`: Original Data to be modelled.
+- `tsteps`: Timesteps on which ode_data was calculated.
+- `prob`: ODE problem that the Neural Network attempts to solve.
+- `loss_function`: Any arbitrary function to calculate loss.
+- `grp_size`: The group size achieved after splitting the ode_data into equal sizes.
+- `continuity_term`: Multiplying factor to ensure continuity of predictions throughout different groups.
+
+!!!note
+		The parameter 'continuity_term' should be a relatively big number to enforce a large penalty whenever the last point of any group doesn't coincide with the first point of next group.
+"""
+function loss_neuralode_helper(p::Array,ode_data::Array,tsteps::Array,prob::ODEProblem,loss_function::Function,grp_size::Integer=5,continuity_term::Integer=100)
```

Comment:
This should probably have "multiple shooting" in its name. Also, have you taken a look at https://github.com/SciML/DiffEqParamEstim.jl/blob/e99fb8233f48be99540dc9bf63ac226bb22c3c22/src/multiple_shooting_objective.jl#L24?

Reply:
Hi, I'll change the name of the function to 'multiple_shooting'. And I took a look at the link now, do you have any suggestions from there? 😅

Reply:
Not really, just added it for your reference